packages/mui-codemod/src/deprecations/dialog-props/dialog-props.test.js

```diff
@@ -0,0 +1,65 @@
+import path from 'path';
+import { expect } from 'chai';
+import { jscodeshift } from '../../../testUtils';
+import transform from './dialog-props';
+import readFile from '../../util/readFile';
+
+function read(fileName) {
+  return readFile(path.join(__dirname, fileName));
+}
+
+describe('@mui/codemod', () => {
+  describe('deprecations', () => {
+    describe('dialog-props', () => {
+      it('transforms props as needed', () => {
+        const actual = transform({ source: read('./test-cases/actual.js') }, { jscodeshift }, {});
+
+        const expected = read('./test-cases/expected.js');
+        expect(actual).to.equal(expected, 'The transformed version should be correct');
+      });
+
+      it('should be idempotent', () => {
+        const actual = transform({ source: read('./test-cases/expected.js') }, { jscodeshift }, {});
+
+        const expected = read('./test-cases/expected.js');
+        expect(actual).to.equal(expected, 'The transformed version should be correct');
+      });
+
+      it('actual.js should not be equal to expected.js', () => {
```

Comment:
content in accordion [actual.js](https://github.com/mui/material-ui/blob/master/packages/mui-codemod/src/deprecations/accordion-props/test-cases/actual.js) file is exactly equal to[ expected.js file](https://github.com/mui/material-ui/blob/master/packages/mui-codemod/src/deprecations/accordion-props/test-cases/expected.js), i belive it's wrong. hence added this test.

Reply:
Sorry I missed this, fixed here: https://github.com/mui/material-ui/pull/40879